tests/integration/targets/vmware_rest_appliance-multi-hosts/playbook.yml

```diff
@@ -0,0 +1,14 @@
+- hosts: localhost
+  gather_facts: no
+
+  tasks:
+    - name: Import eco-vcenter credentials
```

Comment:
you don't need to import it, unless you use vars from this file in a test. Authorization occurs in init-eco.sh

Reply:
fixed :)